test/mailers/application_mailer_test.rb

```diff
@@ -34,7 +34,7 @@ def mail(to = 'nobody@example.com', subject = 'Danger, Will Robinson!')
   end
 
   test 'foreman server header is set' do
-    assert_equal mail.header['X-Foreman-Server'].to_s, 'foreman.some.host.fqdn'
+    assert_equal mail.header['X-Foreman-Server'].to_s, SETTINGS[:fqdn]
```

Comment:
This seems wrong:
https://github.com/theforeman/foreman/blob/develop/app/mailers/application_mailer.rb#L13

Reply:
I was just missing asterisk at except, so you were right, I was trying to solve a wrong bug xD